runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -110,11 +111,15 @@ public Timestamp getTimestamp() {
                              @Nullable final Class<M> mClass,
                              @Nonnull final TableOptions tableOptions)
             throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+        long startTime = System.nanoTime();
         Optional<Timer.Sample> sample = MeterRegistryProvider.getInstance().map(Timer::start);
         Table table =
                 runtime.getTableRegistry().openTable(namespace, tableName, kClass, vClass, mClass, tableOptions);
         corfuStoreMetrics.recordTableCount();
+        int tableSize = table.count();
```

Comment:
Can you document that this count is to force a sync on the table, or maybe just introduce a sync method and underneath you can call size.

Reply:
Actually, we had a discussion and looks they rather want to retain existing behavior, so I'll remove this count().
If someone wants to force a sync they can call count() or another similar getter api on their own.